README.md

```diff
@@ -1,87 +1,71 @@
 
-## Technology used
+This project is part of a larger initiative at Hack for LA around creating a shared housing application, that can be used by organizations across the county to help people transition to living independently and sustainably through host homes, empty bedrooms, and roommate matching for shared housing with lease signing.
 
-### Frontend
-- [TypeScript](https://www.typescriptlang.org/)
-- [React](https://reactjs.org/docs/getting-started.html)
-- [Redux Toolkit](https://redux-toolkit.js.org/)
-- [Material UI](https://material-ui.com/)
-- [Vite](https://vitejs.dev/)
+## Technology Overview
 
-### Backend
-- [Python/Flask](https://flask.palletsprojects.com/en/1.1.x/)
-- [PostgreSQL](https://www.postgresql.org/)
-- [SQLAlchemy](https://www.sqlalchemy.org/)
-- [Docker](https://docs.docker.com/)
-- [Nginx](https://nginx.org/en/docs/)
-- [AWS](https://docs.aws.amazon.com/)
+The HomeUniteUs project is structured as a multi-[docker](https://docs.docker.com/) container application, with secret-protected access to networked resources. The project contains three containers, whose activities are coordinated using the `docker compose` configuration outlined in `docker-compose.yml`. The three containers are:
 
-# How to contribute
+1. `app`: A frontend [React](https://reactjs.org/docs/getting-started.html) app developed using [TypeScript](https://www.typescriptlang.org/).
+   * We use [Redux](https://redux.js.org/) to manage client state, with the [Redux Toolkit](https://redux-toolkit.js.org/) to simplify development.
+   * We use the [Material UI](https://material-ui.com/) component library, for access to high quality UI components designed with accessibility in mind.
+   * We use the [Vite](https://vitejs.dev/) build tool, for fast dev builds and optimized production builds.
+2. `api`: A backend python [Flask](https://flask.palletsprojects.com/en/1.1.x/) REST API, hosted on [AWS](https://docs.aws.amazon.com/).
+   * We use [connexion](https://connexion.readthedocs.io/en/latest/) to simplify our API specification and validation. `connexion` uses a .yaml API specification to map API URLs to specific python functions within our `Flask` app, and will handle routing and parameter passing when an API request is made. `connexion` also generates documentation for our API, and provides a useful user interface (at `{URL}/api/ui`) that can be to easily interact with the API for development purposes.
+   * We use the [SQLAlchemy](https://www.sqlalchemy.org/) SQL toolkit and [Object-Relational Mapper (ORM)](https://en.wikipedia.org/wiki/Object%E2%80%93relational_mapping). This serves as a developer-friendly layer between the python app and the database.
+   * We use [gunicorn](https://gunicorn.org/) as our WSGI server. A WSGI server acts as a communication intermediary between the web server and the `Flask` application, handling client requests and passing them to the application, then returning the application's responses to the client.
```

Comment:
If the `api` description above is updated, then this should also be updated to say "...intermediary between an HTTP proxy and the Home Unite Us API" rather than "`Flask` application".

Reply:
Perfect! Yeah I was intentionally vague because it wasn't clear in my head. Thanks for the help.

Reply:
Corrected in [ebfba50](https://github.com/hackforla/HomeUniteUs/pull/557/commits/ebfba503f84ed549893cd2f68623dc90597fdafe)